test/CMakeLists.txt

```diff
@@ -136,6 +136,17 @@ set(libyul_sources
 )
 detect_stray_source_files("${libyul_sources}" "libyul/")
 
+set(yul_phaser_sources
+    yul_phaser/Chromosome.cpp
```

Comment:
Can you use camelCase for directories, please?

Reply:
Sure. I was following the convention used in `solidity-upgrade` (#7914) but now I see that dirs elsewhere all use camel case. So this probably should be changed in `solidity-upgrade` too.

Reply:
Done.